demo/src/main/java/com/example/demo/service/DislikedVideoService.java

```diff
@@ -0,0 +1,39 @@
+package com.example.demo.service;
+
+import com.example.demo.models.DislikedVideo;
+import com.example.demo.models.UserDataBase;
+import com.example.demo.models.Video;
+import com.example.demo.repo.DislikedVideoRepository;
+import com.example.demo.repo.UserRepository;
+import com.example.demo.repo.VideoRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DislikedVideoService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private VideoRepository videoRepository;
+
+    @Autowired
+    private DislikedVideoRepository dislikedVideoRepository;
+
+    public boolean addToDislikedVideos(String token, String identifier) {
```

Comment:
Can like and dislike same video?
At this point it seems that it would be way easier to have single Like entity with boolean field "isLike" (instead of separate tables for likes and dislikes).